BREAKINGCHANGES_dev.md

```diff
@@ -0,0 +1,7 @@
+# Configuration
```

Comment:
It might be better to simply say:

```md
1. The `enable_winget` configuration entry in the `windows` section has been removed 
   as it won't have issues and thus will be enabled by default.
```

and keep the format simple:)